### ISSUE TITLE
🐞 Bug report: Tabs Widget is not keyboard accessible

### ISSUE THREAD

**Issue by bpbezerra** [elementor/elementor#19103]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The tab widget is not accessible via keyboard navigation

### Steps to reproduce

Insert a tag widget, open the page and try to navigate through the tabs with your keyboard, both using the Tab key or the arrow keys

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.46 (Unix) OpenSSL/1.1.1n
	MySQL version: Source distribution v10.3.29
	PHP Version: 7.4.19
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 80M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0
	Site URL: https://stonetechpcd.corporate.gama.academy
	Home URL: https://stonetechpcd.corporate.gama.academy
	WP Multisite: Yes
	Max Upload Size: 2 MB
	Memory limit: 64M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: pt-BR
	Timezone: America/Sao_Paulo
	Admin Email: marketing@gama.academy
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.5.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: pt_BR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.66 Safari/537.36 Edg/103.0.1264.44

== Network Plugins ==
	Backup
		Version: 1.6.8.6
		Author: BackupGuard

	Branda
		Version: 3.4.7
		Author: WPMU DEV

	Dynamic Visibility for Elementor
		Version: 4.1.2
		Author: Dynamic.ooo

	Elementor
		Version: 3.6.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.2
		Author: Elementor.com

	Insert Headers and Footers
		Version: 1.6.2
		Author: WPBeginner

	Loco Translate
		Version: 2.6.2
		Author: Tim Whitlock

	Mask Form Elementor
		Version: 3.0
		Author: Rodrigo Bogdanowicz

	One User Avatar
		Version: 2.3.9
		Author: One Designs

	Rank Math SEO
		Version: 1.0.92.1
		Author: Rank Math

	Really Simple SSL
		Version: 5.3.2
		Author: Really Simple Plugins

	Redirection
		Version: 5.2.3
		Author: John Godley

	reSmush.it Image Optimizer
		Version: 0.4.2
		Author: reSmush.it

	Safe SVG
		Version: 2.0.2
		Author: 10up

	Wordfence Security
		Version: 7.5.11
		Author: Wordfence

	WordPress Importer
		Version: 0.7
		Author: wordpressdotorg

	WP Fastest Cache
		Version: 1.0.2
		Author: Emre Vona

	WP File Manager
		Version: 7.1.5
		Author: mndpsingh287

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	form-mask.php
		Version: 
		Author: 

== Uso dos elementos ==
	
	wp-page : 1
		accordion : 1
		button : 2
		divider : 4
		heading : 16
		icon-box : 14
		icon-list : 1
		image : 4
		text-editor : 11

== Experimentos do Elementor ==
	Saída DOM otimizada: Ativo por padrão
	Carregamento melhorado de recursos: Ativo por padrão
	Carregamento melhorado de CSS: Ativo por padrão
	Ícones da fonte inline: Inativo por padrão
	Melhorias na acessibilidade: Ativo por padrão
	Importar e exportar Kit de modelos: Ativo por padrão
	Pontos de quebra adicionais personalizados: Ativo por padrão
	Ocultar os widgets nativos do WordPress nos resultados da pesquisa: Ativo por padrão
	: Inativo por padrão
	Flexbox Container: Inativo por padrão
	Default to New Theme Builder: Ativo por padrão
	Tema Hello - Cabeçalho & Rodapé: Ativo por padrão
	Páginas de Aterrissagem: Ativo por padrão
	Amostras de cor: Ativo por padrão
	Widgets favoritos: Ativo por padrão
	Barra superior do administrador: Ativo por padrão
	Page Transitions: Ativo por padrão
	Notes: Ativo por padrão
	Form Submissions: Ativo por padrão
	Scroll Snap: Ativo por padrão

== Registo ==
	
PHP: showing 4 of 4PHP: 2022-07-05 16:26:06 [notice X 299][/bitnami/wordpress/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] O argumento $control_id está obsoleto desde a versão 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-07-05 16:34:47 [notice X 1][/bitnami/wordpress/wp-content/plugins/elementor/core/common/modules/connect/module.php::230] Trying to get property 'email' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-07-06 14:53:44 [notice X 1][/bitnami/wordpress/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-07-06 14:54:14 [notice X 10][/bitnami/wordpress/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] O argumento $control_id está obsoleto desde a versão 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /bitnami/wordpress/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /bitnami/wordpress/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /bitnami/wordpress/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /bitnami/wordpress/wp-content/plugins/dynamic-visibility-for-elementor/class/controls.php(48): Elementor\Controls_Manager -> register_control()
',
)]

JS: showing 1 of 1JS: 2022-07-06 19:35:18 [error X 6][https://stonetechpcd.corporate.gama.academy/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.7:3:897948] Cannot read properties of null (reading \'getBoundingClientRect\') 

== Elementor - Compatibility Tag ==
	

== Elementor Pro - Compatibility Tag ==

**Comment by rami-elementor** [comment]
@bpbezerra 

Try activating the "Accessibility Improvements" experiment,
